### ISSUE TITLE
syntax error 

### ISSUE THREAD

**Issue by DanyalAndriano** [learn-co-curriculum/dsc-1-03-09-accessing-data-within-pandas-lab#1]
Copied directly, this solution code returns a syntax error `print(len(df[df.Total Goals >= 5]))`

**Comment by akelly66** [comment]
This is incorrect, as the question asks for MORE than 5 goals, not 5 or more. Either update the answer or the question to match.
